cmd/kubelet/app/options/options.go

```diff
@@ -23,7 +23,6 @@ import (
 
 	utilfeature "k8s.io/apiserver/pkg/util/feature"
 	"k8s.io/apiserver/pkg/util/flag"
```

Comment:
Thanks! Please keep the `utilflag` one instead, for consistency with the other pkgs that come from util subdirs. Otherwise lgtm.